### PR TITLE
refactor: automate payrequest.show() tests

### PR DESCRIPTION
**Pull request by marcoscaceres**
Now uses web driver.

### CODE REVIEW

File: payment-request/payment-request-show-method.https.html [hunk 1]
```diff
@@ -27,4 +28,40 @@
...
+  button.onclick = async () => {
+    const request1 = new PaymentRequest(defaultMethods, defaultDetails);
+    const request2 = new PaymentRequest(defaultMethods, defaultDetails);
+    const acceptPromise1 = request1.show();
+    const acceptPromise2 = request2.show();
+    await promise_rejects(t, "AbortError", acceptPromise2);
+    await request1.abort();
+    await promise_rejects(t, "AbortError", acceptPromise1);
+  };
+  await test_driver.click(button);
```
[the remark this answers is not in the document]
Comment (marcoscaceres) [reply payment-request/payment-request-show-method.https.html]: oh, awesome! so:

```JS
promise_test(async t => {
   test_driver.bless("payment request to show", async()=>{
     // stuff 
   });
})
```

[the remark this answers is not in the document]
Comment (marcoscaceres) [reply payment-request/payment-request-show-method.https.html]: @lukebjerring, we probably want to do all of them in once go, after this lands.

[the remark this answers is not in the document]
Comment (marcoscaceres) [reply payment-request/payment-request-show-method.https.html]: That is, we  have bunch of these button using tests... could I bug you for review later? 🙏😅

File: payment-request/payment-request-show-method.https.html [hunk 2]
```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
-<!-- Copyright © 2017 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
 <meta charset="utf-8">
 <title>Test for PaymentRequest.show() method</title>
 <link rel="help" href="https://w3c.github.io/browser-payment-api/#show-method">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
```
Comment (lukebjerring) [inline payment-request/payment-request-show-method.https.html]: You'll need to pull in `/resources/testdriver.js` here; currently failing with

> `promise_test: Unhandled rejection with value: object "ReferenceError: test_driver is not defined"`

https://travis-ci.org/web-platform-tests/wpt/jobs/404559383

File: payment-request/payment-request-show-method.https.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukebjerring) [reply payment-request/payment-request-show-method.https.html]: Sure - this looks good as-is for a refactor, let's follow up with the bless stuff in another PR.

**Review by lukebjerring — CHANGES_REQUESTED** [review]

File: payment-request/payment-request-show-method.https.html (hunk 2 above)
Comment (lukebjerring) [reply payment-request/payment-request-show-method.https.html]: And, `testdriver-vendor.js` (empty, but enforced by a lint rule)

Comment (marcoscaceres) [reply payment-request/payment-request-show-method.https.html]: Ah, don't know how i missed that :/

**Comment by marcoscaceres** [comment]
Filed bug to refactor... only need to refactor two files, so should be quick:
https://github.com/web-platform-tests/wpt/issues/12082

**Review by lukebjerring — APPROVED** [review]

**Pull request merged**